### ISSUE TITLE
Any change of the Deepin DE coming to Clear Linux?

### ISSUE THREAD

**Issue by ZVNexus** [clearlinux/distribution#1798]

**Comment by ahkok** [comment]
Unlikely. No resources for additional custom desktop environments. I wouldn't mind it, but, if nobody in the community wants to contribute these packages, it's likely not going to happen.

**Comment by ahkok** [comment]
One of the big problems I have is that Deepin requires a forked mutter. One mutter is already terrible enough, we really do not want to maintain 2 of these.

**Comment by ketonik** [comment]
>ahkok, wouldn't elementary OS guys be a perfect fit for Intel? Those guys are dedicated to UI/UX and produce beautiful desktop environment, way more polished than anything else out there. Intel guys a dedicated to focus on their objective with Clear Linux, performance, security etc.

It's a bit of a shame elementary OS has Ubuntu underpinning it instead of Clear Linux, Intelix ;-)

**Comment by ZVNexus** [comment]
Elementary doesn't have blur though, so instantly inferior /s

**Comment by ketonik** [comment]
Blur as in an effect?

I dunno, I check out deepin, my conclusion is that it's just a Chinese copy of Windows UI/UX with blur (opacity), like most Linux DE, why they all copy Windows, ie start button, task bar down bottom, to me Windows has always been the most illogical unintuitive GUI.

**Comment by ZVNexus** [comment]
Yeah IMO opacity and blur and practically all windows just looks really good. Only a few DEs I know that can do it, and only one out of the box, Deepin. KDE exists but far too much configuration to get that to a point where I actually like it

**Comment by ketonik** [comment]
Oh I understand you, i've had this on macOS for years, actually I shouldn't even post suggestions or preferences on DE, none of my Linux installs even have a DE, all console based servers. I use macOS on desktop and so Linux Desktop could never satisfy my requirements, looking at the the history, it hasn't evolved much since 1997, actually de-evolved, I don't see it changing much.

Deepin certainly looks pretty and visually attractive for sure.

**Comment by isaakhh** [comment]
Is it absolutely certain, that it will not be added? And if so, can it be added some other way? I wanted to compile it manually but could not even find the source code for the whole Desktop Environment.
